apps/settings/locales/settings.en-US.properties

```diff
@@ -1301,6 +1301,7 @@ improveBrowserOSTitle=Improve {{brandShortName}}
 sendMozillaFeedback=Send Mozilla Feedback
 performanceAndUsageData=Performance and Usage Data
 sharePerformanceAndUsageData=Submit data
+openAppStats=Open App Statistics
```

Comment:
We should make title consistent, I can change mine to `App Statistics` but I'm not sure its the best title. `App Usage Statistics` maybe? We've got App Statistics on the Marketplace that are something else. Not sure at this point :)